extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -157,7 +162,7 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
    * @private
    */
   onShoppingDataUpdate_(shoppingData) {
-    if (!shoppingData.activeProductData || !this.isOnActivePage_()) {
+    if (!this.isOnActivePage_() || !shoppingData.activeProductData) {
```

Comment:
Out of curiosity, what is this edit for?

Reply:
See your own comment here for the explanation:
https://github.com/ampproject/amphtml/pull/38041#discussion_r846207964

Reply:
That comment was about moving the check lower in the function in [this commit](https://github.com/ampproject/amphtml/pull/38041/commits/89e78968ff9af41a44a389d970a09ffa737e81d3) which caused `updateTemplate_` to run on every page.

I'm asking why the ternary is flipped. Is there a performance difference?

Reply:
Ok, then there wouldn't be a performance difference then, I can change it back.